cmd/release-notes/main.go

```diff
@@ -149,8 +149,21 @@ func init() {
 	cmd.PersistentFlags().StringVar(
 		&opts.Format,
 		"format",
-		util.EnvDefault("FORMAT", "markdown"),
-		"The format for notes output (options: markdown, json)",
+		util.EnvDefault("FORMAT", notes.FormatOutputMarkdown),
+		fmt.Sprintf("The format for notes output (options: %s)",
+			strings.Join([]string{
+				notes.FormatOutputJSON,
+				notes.FormatOutputMarkdown,
```

Comment:
Do you think it would be useful to just add another format `notes.FormatOutputTemplate` instead of adding another CLI option? This way we could re-use the output path as well.

Reply:
Yep. That makes sense to me.

Reply:
Revised in next commit.